### PR TITLE
CON-1562 Updated readme file to prototype CI template

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,14 +134,3 @@ Database project containing setup in order to the create the QnA Database
 	
 - projects/{subfolder}/sections
 	- Holds QnAData for each WorkflowSection
-	
-_NOTE:_ To publish the database to your local SQL Server, you need to ensure that `ProjectPath` contains the full path.
-- {drive}:\{project-folders}\das-qna-api\src\SFA.DAS.QnA.Database\
-- For example: *C:\projects\efsa\das-qna-api\src\SFA.DAS.QnA.Database\*
```
Comment (martin-hornsby) [inline README.md]: Unless this works out of the box (which I am doubtful of), can you keep this in?

File: README.md [hunk 2]
```diff
@@ -1,46 +1,95 @@
...
+- Update Configuration SFA.DAS.QnA.API_1.0, Data { "SqlConnectionstring":"Server={{local instance name}};Initial Catalog={{database name}};Trusted_Connection=True;" }
 
-#### SFA.DAS.QnA.Api.Types
-Common types to interact with the QnA Api
-- Published to Nuget: https://www.nuget.org/packages/SFA.DAS.QnA.Api.Types
+##### Run the solution
 
-#### SFA.DAS.QnA.Api.Views
-Common views to display questions
-- Published to Nuget: https://www.nuget.org/packages/SFA.DAS.QnA.Api.Views
```
Comment (martin-hornsby) [inline README.md]: I'd keep these in for information so that people know some stuff is published to NuGet

File: README.md (hunk 1 above)
Comment (ChrisJWoodcock) [reply README.md]: Is it still there - Just that I've moved it to the more standard database publish section above

File: README.md (hunk 2 above)
Comment (ChrisJWoodcock) [reply README.md]: When you view this as a Markdown file - all external items are at the top with badges - these have Nuget build badges giving their latest version and a link to the nuget package.

**Pull request merged**